configs/components/puppet.rb

```diff
@@ -4,6 +4,7 @@
   pkg.build_requires "ruby-#{settings[:ruby_version]}"
   pkg.build_requires "facter"
   pkg.build_requires "hiera"
+  pkg.build_requires "rubygem-win32-dir"
```

Comment:
In the interest of getting the component pipeline unblocked, I vote for that last suggestion.